src/org/daisy/dotify/formatter/impl/row/SegmentProcessor.java

```diff
@@ -1102,12 +1097,13 @@ private String getLeftIndentWithLabel(String listLabel, int indent) {
         private Optional<RowImpl> continueRow(
             RowInfo row,
             BrailleTranslatorResult btr,
-            int nextRowIndent,
             int rightIndentIfNotLastRow,
             String mode,
             LineProperties lineProps
         ) {
-            RowImpl ret = null;
+            if (!hasCurrentRow()) {
+                throw new RuntimeException("Error in code.");
```

Comment:
I'd prefer a better explanation.

Reply:
This is basically just to assert that this method is never called when `!hasCurrentRow()`. If this assertion fails it means there is a bug in the code. I'll add some more explanation.

Reply:
Added some comments.